embedded/projects/SH-Xpresso-LPC54102/sources/app/sensoracq_t.c

```diff
@@ -129,7 +130,7 @@ static void SensorDataHandler(InputSensor_t sensorId, uint32_t timeStamp)
         if ((sMagDecimateCount++ % MAG_DECIMATE_FACTOR) == 0 )
         {
             /* Read mag Data - reading would clear interrupt also */
-            Mag_ReadData( &magData );
+            Driver_Mag.ReadData(&magData,0);
```

Comment:
Since this parameter wasn't being used, I had simply passed 0. However I'll correct it in next commit.